README.md

```diff
@@ -1 +1,10 @@
 # hw01
+## Omair Gill's Biography
+I was born and raised in *San Mateo, California*. I attended my undergraduate studies at University of California, Davis and double majored in **Economics and International Relations**. I am currently a second year MPP student at the Harris School of Public Policy. Please see [my Linkedin profile] (https://www.linkedin.com/in/omair-gill-1b0038164) for more information on my professional background. My hobbies include:  
+
+* Cooking
+* Running
+* Watching basketball
+
+Here is a photo of my idol and best player in the league, Steph Curry.
+![](https://upload.wikimedia.org/wikipedia/commons/thumb/2/27/Stephen_Curry_Shooting_%28cropped%29.jpg/800px-Stephen_Curry_Shooting_%28cropped%29.jpg)
```

Comment:
Hi Omair, love the way you organised your biography, keeping it simple. Not to copy Dr. Soltoff's comment, but the picture could have been smaller maybe?

Reply:
Yes, agreed!

Reply:
@sharanyabashyam @ogill-stuart no easy method to automatically shrink an image using Markdown syntax. Markdown is simple and quick to learn, but the drawback is that it doesn't not allow for a lot of customization or complexity 😞